src/draw/sw/lv_draw_sw.c

```diff
@@ -289,6 +289,8 @@ static int32_t evaluate(lv_draw_unit_t * draw_unit, lv_draw_task_t * task)
     return 0;
 }
 
+extern int flush_cnt;
```

Comment:
I think this should go to top of file, so we can see immediately what variables are used.